aws/table_aws_securityhub_finding.go

```diff
@@ -0,0 +1,422 @@
+			{
+				Name:        "finding_provider_fields",
+				Description: "In a BatchImportFindings request, finding providers use FindingProviderFields to provide and update their own values for confidence, criticality, related findings, severity, and types.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "malware",
+				Description: "A list of malware related to a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network",
+				Description: "The details of network-related information about a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network_path",
+				Description: "Provides information about a network path that is relevant to a finding. Each entry under NetworkPath represents a component of that path.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "note",
+				Description: "A user-defined note added to a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "patch_summary",
+				Description: "Provides an overview of the patch compliance status for an instance against a selected compliance standard.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "process",
+				Description: "The details of process-related information about a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "product_fields",
+				Description: "A data type where security-findings providers can include additional solution-specific details that aren't part of the defined AwsSecurityFinding format.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "related_findings",
+				Description: "A list of related findings.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "remediation",
+				Description: "A data type that describes the remediation options for a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "resources",
+				Description: "A set of resource data types that describe the resources that the finding refers to.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "severity",
+				Description: "A finding's severity.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "threat_intel_indicators",
+				Description: "Threat intelligence details related to a finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "user_defined_fields",
+				Description: "A list of name/value string pairs associated with the finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "vulnerabilities",
+				Description: "Provides a list of vulnerabilities associated with the findings.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			/// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: "A finding's title.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "akas",
```

Comment:
Do security hub findings have actual ARNs in AWS? If not, we can treat these like we do logs in tables, and not create the `akas` column